doc/lua_api.txt

```diff
@@ -4883,12 +4883,18 @@ This is basically a reference to a C++ `ServerActiveObject`
   `set_look_vertical`.
 * `set_look_yaw(radians)`: sets look yaw - Deprecated. Use
   `set_look_horizontal`.
-* `get_breath()`: returns players breath
-* `set_breath(value)`: sets players breath
+* `get_breath()`: returns player's breath
+* `set_breath(value)`: sets player's breath
     * values:
         * `0`: player is drowning
         * max: bubbles bar is not shown
         * See Object properties for more information
+* `set_fov(fov)`: sets player's FOV
+    * `fov`: Accepts values from 1 to 160, in degrees.
+    * Use `clear_fov` to reset player's FOV to its default
```

Comment:
Why not just specify that `0` (or negative values) will clear the FOV overwrite? `clear_fov` will be superfluous in that case.

Reply:
I agree, 0 is good.

Reply:
Alright, removed `clear_fov`